README.md

```diff
@@ -16,7 +16,7 @@ For now the main supported country is France.
 
 OpenFisca is a modular project. Depending on your goals, you will install and interact with one or several of the OpenFisca Components.
 
-[![OpenFisca schema](https://cdn.rawgit.com/openfisca/openfisca-doc/eb7b9ce6f41d7ab2cb7bac08738cfb67d58c0c36/architecture.svg)](https://github.com/openfisca/openfisca-doc/blob/bird_eye_view/architecture.svg)
+[![OpenFisca schema](https://cdn.rawgit.com/openfisca/openfisca-doc/0207413e/architecture.svg)](https://github.com/openfisca/openfisca-doc/blob/master/architecture.svg)
```

Comment:
Yes, I know 🙂 I just don't think the current traffic on the OpenFisca documentation is going to be considered “excessive” :wink: and if we ever run into issues, we can always go back to the static version.